examples/lighting-app/infineon/cyw30739/include/CHIPProjectConfig.h

```diff
@@ -34,13 +34,13 @@
 #define CHIP_DEVICE_CONFIG_DEVICE_VENDOR_NAME "Infineon"
 
 /* The VendorID attribute of the Basic cluster. */
-#define CHIP_DEVICE_CONFIG_DEVICE_VENDOR_ID 0xFFF1
+#define CHIP_DEVICE_CONFIG_DEVICE_VENDOR_ID 0xFFF2
```

Comment:
Both of these are test IDs ... is this change needed? Does it have to do with DAC/PAI/CD?

Reply:
Yes, it's needed.
CYW30739 example apps use the test DAC with VID 0xFFF2 by default.
Test IDs have to be identical to IDs in the test DAC for chip-tool pairing.